.github/workflows/build.yml

```diff
@@ -307,12 +308,11 @@ jobs:
       - name: Install Packer plugins
         run: packer init .
       - name: Create machine image
-        # This runs through the AMI creation process but does not
-        # actually create an AMI
         run: |
```

Comment:
With this change we really need some kind of auto-cleanup on AMIs. Since this runs on `push` we are liable to have _a lot_ of AMIs created.

Reply:
Definitely agree with this - we should revisit https://github.com/cisagov/cool-system/issues/198 and invest some time in getting that done.